packages/neherlab-covid-19-scenarios/src/basel-api.ts

```diff
@@ -89,61 +144,76 @@ export interface ExposedCurrentData {
   overflow: Record<string, number>
 }
 
-export interface CumulativeData {
+export interface ExposedCumulativeData {
   recovered: Record<string, number>
   hospitalized: Record<string, number>
   critical: Record<string, number>
   fatality: Record<string, number>
 }
 
+export interface TimePoint {
+  t: number
+  y: number
+}
+
+export type TimeSeries = TimePoint[]
+
 // This defines the user-facing data structure
 export interface ExportedTimePoint {
   time: number
   current: ExposedCurrentData
-  cumulative: CumulativeData
+  cumulative: ExposedCumulativeData
 }
 
 export interface ModelFracs {
-  severe: Record<string, number>
-  critical: Record<string, number>
-  fatal: Record<string, number>
-  isolated: Record<string, number>
+  severe: number[]
+  critical: number[]
+  fatal: number[]
+  isolated: number[]
 }
 
 export interface ModelRates {
   latency: number
-  infection: (t: Date) => number
-  recovery: Record<string, number>
-  severe: Record<string, number>
-  discharge: Record<string, number>
-  critical: Record<string, number>
-  stabilize: Record<string, number>
-  fatality: Record<string, number>
-  overflowFatality: Record<string, number>
+  infection: (t: number) => number
+  recovery: number[]
+  severe: number[]
+  discharge: number[]
+  critical: number[]
+  stabilize: number[]
+  fatality: number[]
+  overflowFatality: number[]
 }
 
 export interface ModelParams {
-  ageDistribution: Record<string, number>
-  importsPerDay: Record<string, number>
+  ageDistribution: AgeDistributionDatum[]
+  importsPerDay: number[]
   timeDelta: number
   timeDeltaDays: number
   populationServed: number
   numberStochasticRuns: number
   hospitalBeds: number
-  ICUBeds: number
+  icuBeds: number
```

Comment:
Don't know if this is important, but `scenarios.json` has ICUBeds, unless that has changed?

Reply:
See https://github.com/neherlab/covid19_scenarios/blob/1.4.4/src/assets/data/scenarios.json.